test/kpod_stats.bats

```diff
@@ -6,6 +6,10 @@ function teardown() {
     cleanup_test
 }
 
+function setup() {
+    copy_images
```

Comment:
I don't think we end up using any images other than `redis:alpine` in this file, so this won't save us time.